pkg/proto/backup_schedule_service.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package ydbcp;
+option go_package = "github.com/ydb-platform/ydbcp/pkg/proto;ydbcp";
+
+import "backup_schedule.proto";
+import "google/protobuf/duration.proto";
+
+service BackupScheduleService {
+    rpc CreateBackupSchedule (CreateBackupScheduleRequest) returns (BackupSchedule);
+    rpc UpdateBackupSchedule (UpdateBackupScheduleRequest) returns (BackupSchedule);
+    rpc GetBackupSchedule (GetBackupScheduleRequest) returns (BackupSchedule);
+    rpc ListBackupSchedules (ListBackupSchedulesRequest) returns (ListBackupSchedulesResponse);
+
+    rpc ActivateBackupSchedule (ToggleBackupScheduleRequest) returns (BackupSchedule);
+    rpc DeactivateBackupSchedule (ToggleBackupScheduleRequest) returns (BackupSchedule);
+
+    //QUESTION: do we need DeleteBackupSchedule?
+//    rpc DeleteBackupSchedule (DeleteBackupScheduleRequest) returns (DeleteBackupScheduleResponse);
+}
+
+message CreateBackupScheduleRequest {
+    //fields of MakeBackupRequest to schedule
+    string container_id = 1;
+    string database_name = 2;
+    string endpoint = 3;
+    repeated string source_paths = 4;  // [(size) = "<=256"];
+    repeated string source_paths_to_exclude = 5;  // [(size) = "<=256"];
+
+    //QUESTION: do we need schedule_name?
+    string schedule_name = 6;
+    BackupScheduleSettings schedule_settings = 7;
+}
+
+message ListBackupSchedulesRequest {
+    string container_id = 1;
+    string database_name_mask = 2;
+
+    // The maximum number of results per page that should be returned. If the number of available
+    // results is larger than `page_size`, the service returns a `next_page_token` that can be used
+    // to get the next page of results in subsequent ListBackupSchedules requests.
+    // Acceptable values are 0 to 1000, inclusive. Default value: 100.
+    uint32 page_size = 1000; // [(value) = "0-1000"];
+
+    // Page token. Set `page_token` to the `next_page_token` returned by a previous ListBackups
+    // request to get the next page of results.
+    string page_token = 1001; // [(length) = "<=100"];
+}
+
+message ListBackupSchedulesResponse {
+    repeated BackupSchedule schedules = 1;
+    string next_page_token = 2;
+}
+
+//QUESTION: do we need UpdateBackupScheduleRequest?
+//it is almost the same as create.
+message UpdateBackupScheduleRequest {
+    string id = 1;
+    //QUESTION: or maybe we leave out all of this and
+    //allow to change only BackupScheduleSettings and name
+    string container_id = 2;
```

Comment:
for what we need container_id?

Reply:
to know in which container we should create backups